compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/test/mma_attrs.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: iree-opt %s | iree-opt | FileCheck %s
```

Comment:
it's to check round-trip parsing + printing. Doesn't really matter here so I'll drop it, but I've caught bugs before by doing this because sometimes there is inconsistency between the parser and the printer.